tests/unit/TodoList2.spec.ts

```diff
@@ -67,11 +67,16 @@ describe('TodoList - common expansion panel header tests', () => {
 
     const sandbox = sinon.createSandbox()
     const fetchAffiliationInvites = sandbox.stub(AuthServices, 'fetchAffiliationInvitations')
-    // feature flag override
-    const featureFlag = sandbox.stub(utils, 'GetFeatureFlag')
 
-    featureFlag.withArgs('enable-affiliation-invitation-request-access')
-      .returns(true)
+    // feature flag override
+    vi.spyOn(utils, 'GetFeatureFlag').mockImplementation(
```

Comment:
This test was failing:
![failed test todolist](https://github.com/bcgov/business-filings-ui/assets/122301442/f75884df-8cb3-4a2a-b98b-d42d5b473c76)

After spending some time on it, I found out the feature flag override wasn't working properly. Hence, the todo items list is always being 0. By mocking the feature flag like this, it'll fix the test.